### PR TITLE
docs: update taiko branding guide

### PR DESCRIPTION
**Pull request by nikolay44u**
Updated Branding pages with images of logos, links and font licenses.

### CODE REVIEW

File: packages/branding/README.md [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
 
-For naming, we use Taiko or Taiko Labs. For the handle, we refer to Taiko with `taikoxyz`, which is where you can find us (ie. twitter: @taikoxyz, github: @taikoxyz). Please maintain this convention.
+For naming, we use Taiko or Taiko Labs. For our social media handle, we refer to Taiko with `taikoxyz`, which is where you can find us (ie. twitter: [@taikoxyz](https://twitter.com/taikoxyz), github: [@taikoxyz](https://github.com/taikoxyz)). Please maintain this convention.
 
 ## Logos
 
-Use SVG and PNG where you can, as the image quality is better.
+Downloabable vector assets of the Taiko logo:
+
+<img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logo/SVG/Taiko_Token_Fluo-on-Black.svg" width="75px"> &nbsp;&nbsp;&nbsp; <img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logo/SVG/Taiko_Token_White-on-Black.svg" width="75px"> &nbsp;&nbsp;&nbsp; <img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logo/SVG/Taiko_Logo_White-on-Fluo.svg" width="75px">
```
[the remark this answers is not in the document]
Comment (nikolay44u) [reply packages/branding/README.md]: Included primary logo in the preview.

File: packages/branding/README.md [hunk 2]
```diff
@@ -1,11 +1,42 @@
...
+[Browse Logos](https://github.com/taikoxyz/taiko-mono/tree/main/packages/branding/Logo)
+
+Please use SVG or PNG wherever possible, as the image quality is better.
+
+## Logotypes
+
+Downloabable vector assets of the Taiko logotype:
+
+
+<img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logotype/SVG/Taiko_Logotype_Horiz_2_Fluo_White.svg" width="75px"> &nbsp;&nbsp;&nbsp; <img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logotype/SVG/Taiko_Logotype_Square_Fluo_White.svg" width="75px">
```
[the remark this answers is not in the document]
Comment (nikolay44u) [reply packages/branding/README.md]: Included primary logo in the preview.

File: packages/branding/README.md [hunk 3]
```diff
@@ -1,11 +1,42 @@
...
+
+Taiko uses Oxanium as the global font across its identity. The font is licensed under the [Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL). Its specimen and font family can be accessed at: https://fonts.google.com/specimen/Oxanium
+
+## Colors
+
+We use these primary colors and their shades across our brand and products.
+
+Fluo <code>#fc0fc0</code>  
+Black <code>#000000</code>    
+Navy <code>#000018</code>
```
[the remark this answers is not in the document]
Comment (nikolay44u) [reply packages/branding/README.md]: Removed navy from the preview and updated brandbook pdf file with this change.

File: packages/branding/README.md [hunk 4]
```diff
@@ -1,11 +1,41 @@
-# Taiko branding
+# Taiko Brand Resources
```
Comment (d1onys1us) [inline packages/branding/README.md:1]:
```suggestion
# Taiko brand guide
```
ref: https://learn.microsoft.com/en-us/style-guide/capitalization
also, to make it more actionable

File: packages/branding/README.md [hunk 5]
```diff
@@ -1,11 +1,41 @@
...
 
-This package contains all you need for Taiko branding.
+This package contains all the branding assets for Taiko. 
+
+Included: Logos, Logotypes, Backgrounds, Backdrops and Mockups.
 
 ## Naming
 
-For naming, we use Taiko or Taiko Labs. For the handle, we refer to Taiko with `taikoxyz`, which is where you can find us (ie. twitter: @taikoxyz, github: @taikoxyz). Please maintain this convention.
+For naming, we use Taiko or Taiko Labs. For our social media handle, we refer to Taiko with `taikoxyz`, which is where you can find us (ie. twitter: [@taikoxyz](https://twitter.com/taikoxyz), github: [@taikoxyz](https://github.com/taikoxyz)). Please maintain this convention.
```
Comment (d1onys1us) [inline packages/branding/README.md:9]:
```suggestion
Use `Taiko` or `Taiko Labs` for the name.

Use `taikoxyz` for the social handle (e.g., Twitter: [@taikoxyz](https://twitter.com/taikoxyz), GitHub: [@taikoxyz](https://github.com/taikoxyz)).
```
ref: (active voice) https://learn.microsoft.com/en-us/style-guide/global-communications/writing-tips
ref: (don't use we) https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we

File: packages/branding/README.md [hunk 6]
```diff
@@ -1,11 +1,41 @@
...
 
 ## Logos
 
-Use SVG and PNG where you can, as the image quality is better.
+Downloabable vector assets of the Taiko logo:
```
Comment (d1onys1us) [inline packages/branding/README.md:13]:
```suggestion
Download vector assets of the Taiko logo:
```
use actionable phrase

File: packages/branding/README.md [hunk 7]
```diff
@@ -1,11 +1,41 @@
...
+
+<img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logo/SVG/Taiko_Logo_Fluo.svg" width="75px"> &nbsp;&nbsp;&nbsp; <img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logo/SVG/Taiko_Token_Fluo-on-Black.svg" width="75px"> &nbsp;&nbsp;&nbsp; <img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logo/SVG/Taiko_Token_White-on-Black.svg" width="75px"> &nbsp;&nbsp;&nbsp; <img src="https://github.com/taikoxyz/taiko-mono/blob/main/packages/branding/Logo/SVG/Taiko_Logo_White-on-Fluo.svg" width="75px">
+
+[Browse Logos](https://github.com/taikoxyz/taiko-mono/tree/main/packages/branding/Logo)
+
+Please use SVG or PNG wherever possible, as the image quality is better.
```
Comment (d1onys1us) [inline packages/branding/README.md:19]:
```suggestion
Please use SVG or crushed PNG wherever possible for improved image quality and performance.
```
nitpick

File: packages/branding/README.md [hunk 8]
```diff
@@ -1,11 +1,41 @@
...
+
+## Logotypes
+
+Downloabable vector assets of the Taiko logotype:
```
Comment (d1onys1us) [inline packages/branding/README.md:23]:
```suggestion
Download vector assets of the Taiko logotype:
```
use actionable phrase

File: packages/branding/README.md [hunk 9]
```diff
@@ -1,11 +1,41 @@
...
+
```
Comment (d1onys1us) [inline packages/branding/README.md:24]:
```suggestion
```

File: packages/branding/README.md [hunk 10]
```diff
@@ -1,11 +1,41 @@
...
+Taiko uses Oxanium as the global font across its identity. The font is licensed under the [Open Font License](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL). Its specimen and font family can be accessed at: https://fonts.google.com/specimen/Oxanium
+
+## Colors
+
+We use these primary colors and their shades across our brand and products.
+
+Fluo <code>#fc0fc0</code>  
+Black <code>#000000</code>    
+White <code>#ffffff</code>
```
Comment (d1onys1us) [inline packages/branding/README.md:41]:
```suggestion
Primary colors and shades:
| Color | Hex code  |
|-------|-----------|
| Fluo  | `#fc0fc0` |
| Black | `#000000` |
| White | `#ffffff` |
```

**Review by dionysuzx — COMMENTED** [review]
small comments, which you can click on to automatically apply. thanks for your contribution!

**Review by dionysuzx — APPROVED** [review]

**Review by dantaik — APPROVED** [review]

**Pull request merged**